packages/ethereum/deploy/07_HoprNetworkRegistry.ts

```diff
@@ -7,21 +8,29 @@ const main = async function (hre: HardhatRuntimeEnvironment) {
 
   const environmentConfig = PROTOCOL_CONFIG.environments[environment]
   const deployer = await getNamedAccounts().then((o) => ethers.getSigner(o.deployer))
+  const inProd = environmentConfig['network_id'] == 'xdai'
```

Comment:
Should we instead use a configuration in the environments file?

Reply:
we should, I am addressing that in this PR https://github.com/hoprnet/hoprnet/pull/3809 - `environmentConfig['network_id'] == 'xdai'` is used in multiple files currently